pkg/kube/kube.go

```diff
@@ -312,6 +312,10 @@ func (kc *ClientSet) DeploymentIsRunning(name, namespace string) error {
 	return structured.DeploymentIsRunning(kc.KubeInterface, name, namespace)
 }
 
+func (kc *ClientSet) ValidateConfigMapData(name, namespace, key, value string) error {
+	return structured.ConfigMapDataHasKeyAndValue(kc.KubeInterface, name, namespace, key, value)
```

Comment:
Lets use the same name for the ClientSet method, so name `ValidateConfigMapData` as `ConfigMapDataHasKeyAndValue`